.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: "Build Packages"
```

Comment:
What does "Build Packages" mean? What packages?

Reply:
Packages are taken from commit message: i.e. `hello: 1.0.0 -> 2.0.0`. This is also how ofborg works now.

Reply:
Packages described in a PR title as ofborg builds them